viewing_party/party.py

```diff
@@ -1,23 +1,119 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    '''
+    Create a dict containing info about one movie, with "title", "genre", and "rating", as keys, mapped to values passed in as arguments.
+    '''
+    if not title or not genre or not rating:
+        return None
 
+    movie = {}
+    movie["title"] = title
+    movie["genre"] = genre
+    movie["rating"] = rating

```

Comment:
This works well! Consider making it a bit more concise by initializing the whole dictionary at once:
```
new_movie = {
        "title": title,
        "genre": genre,
        "rating": rating
}
```